src/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "host": "localhost",
+    "port": "8888"
```

Comment:
port is a number not a string, "8888" should be 8888 without the quotes. Otherwise we get an error when using the type defined in types.ts

Reply:
my bad! i fixed it